components/todo-input.js

```diff
@@ -1,20 +1,110 @@
-import React from "react";
+import React, { useState } from 'react';
+
 import styled from "styled-components";
 
-export default function TodoInput() {
+export default function TodoInput({addTodo}){
+
+  const [newTodo, setNewTodo] = useState({});
+
+  const changeInputData = (e) => {
+    setNewTodo({...newTodo, [e.target.name]:e.target.value});
+  }
```

Comment:
```suggestion
export default function TodoInput({onSubmit}){
  const [newTodo, setNewTodo] = useState({});

  const handleFormChange = (e) => {
    setNewTodo({...newTodo, [e.target.name]:e.target.value});
  }
```

1. ```newTodo``` 네이밍 좋네요 👍 
2. props를 통해 함수를 넘겨줄 때 ```addTodo```로 보내는 것보다 ```onSubmit```으로 보내는 것이 더 적절합니다. ```pages/index```의 리뷰를 확인해주세요!
3. ```changeInputData```보다는 ```handleFormChange```라는 네이밍이 더 적절합니다. [참고](https://blog.sonim1.com/220#%EC%A2%80-%EB%8D%94-%EB%B3%B5%EC%9E%A1%ED%95%9C-%EB%84%A4%EC%9D%B4%EB%B0%8D-more-complicated-naming)
Input의 value가 바뀐 것보다 제출할 Form의 값이 바뀌었다는 사실이 더 중요하기 때문에 ```InputData```를 ```Form```으로 바꿔주시면 좋을 것 같아요!